test/images/image-util.sh

```diff
@@ -116,12 +116,21 @@ docker_version_check() {
 push() {
   docker_version_check
   TAG=$(<"${IMAGE}"/VERSION)
+  if [[ -f ${IMAGE}/PREVIOUS_VERSION ]]; then
```

Comment:
Let us not create any more complexity for a single one use case, instead just bump the current version to next version and push it. Later you can modify the `manifest.go` file the version you want to refer.

Reply:
If we bump the version right here, we would have to change the version in manifest.go for every image under test/images whenever there's an update. Is that something that we'd want to do for every image push?

Reply:
yes, that is expected and that is how we maintain the list of images used in e2e tests.

Reply:
Ok thanks for confirming. Reverted this file and simply changed manifest.go to point to the updated version.